ParabolInc/parabol
# chore(share-summary): share summary dialog UI

Fixes #8232 

Fixes #8241

Loom: https://www.loom.com/share/4aafbd13d68a48cf80a74586a6942b82

<img width="473" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/daa7ad29-a93e-4222-b860-6da1caba1e9d">


**How to test:** 

- Follow testing steps from PR to see the dialog https://github.com/ParabolInc/parabol/pull/8331
- Also disable slack integration in team settings, check that connect button works from the dropdown
